cou/steps/plan.py

```diff
@@ -369,6 +370,11 @@ def _get_pre_upgrade_steps(analysis_result: Analysis, args: CLIargs) -> list[Pre
     :rtype: list[PreUpgradeStep]
     """
     steps = [
+        PreUpgradeStep(
+            description="Check application vault is not sealed",
```

Comment:
The wording could be improved a bit:

```suggestion
            description="Verify vault application is unsealed",
```

Reply:
Updated.